locations/spiders/tmobile_pl.py

```diff
@@ -0,0 +1,29 @@
+from scrapy import Spider
+
+from locations.dict_parser import DictParser
+from locations.hours import DAYS_PL, OpeningHours
+
+
+class TMobilePLSpider(Spider):
+    name = "tmobile_pl"
+    item_attributes = {"brand": "T-Mobile International", "brand_wikidata": "Q327634"}
```

Comment:
Probably the following, we follow OSM and NSI, not Wikidata for the brand strings.

```suggestion
    item_attributes = {"brand": "T-Mobile", "brand_wikidata": "Q327634"}
```